static/thing.js

```diff
@@ -0,0 +1,21 @@
+class Thing {
```

Comment:
The Thing class is supposed to represent the object itself right (i.e. Car #1)? Should it also contain an id or name to it?